src/main/resources/com/uber/jenkins/phabricator/HarbormasterConduitNotifier/config.jelly

```diff
@@ -0,0 +1,6 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <f:entry title="Build Target" field="buildTarget"
+           description="Name of the env variable containering the Build Target PHID">
```

Comment:
Ah, I meant elsewhere we assume/require that the parameter is called PHID. I'm open to making it an option, but would prefer to keep the default `PHID`. This also becomes less of an issue if we merge this notifier into the other (and use harbormaster HTTP api instead of arc).